docs/platforms/javascript/common/session-replay/troubleshooting.mdx

```diff
@@ -26,13 +26,15 @@ redirect_from:
 
 This guide aims to extend the <PlatformLink to="/troubleshooting/">main troubleshooting</PlatformLink> guide by covering Replay-specific scenarios.
 
+<PlatformSection notSupported={["javascript.deno", "javascript.bun", "javascript.wasm", "javascript.capacitor", "javascript.electron"]}>
```

Comment:
Good catch @c298lee!

(There's currently a bug, deno/bun/wasm are marked as not supported at the top level, but it's not working)